sklearn/preprocessing/data.py

```diff
@@ -2579,6 +2579,150 @@ def quantile_transform(X, axis=0, n_quantiles=1000,
                          " axis={}".format(axis))
 
 
+class BoxCoxTransformer(BaseEstimator, TransformerMixin):
+    """Apply the Box-Cox transform featurewise to make data more Gaussian-like.
+
+    The Box-Cox transformation is a monotonic transformation that is
+    applied to make data more Gaussian-like. This is useful for solving
+    modeling issues related to heteroscedasticity (non-constant variance),
+    or other situations where normality is desired.
+
+    Box-Cox requires input data to be strictly positive. The optimal value
+    for the parameter, lambda, is estimated through maximum likelihood.
+
+    Read more in the :ref:`User Guide <preprocessing_transformer>`.
+
+    Parameters
+    ----------
+    copy : boolean, optional, default=True
+        Set to False to perform inplace computation.
+
+    Attributes
+    ----------
+    lambdas_ : array of float, shape (n_features,)
+        The parameters of the Box-Cox transformation for the selected features.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.preprocessing import BoxCoxTransformer
+    >>> boxcox = BoxCoxTransformer()
+    >>> data = [[1, 2], [3, 2], [4, 5]]
+    >>> print(boxcox.fit(data))
+    BoxCoxTransformer(copy=True)
+    >>> print(np.round(boxcox.lambdas_, 3))
+    [ 1.052 -2.345]
+    >>> print(np.round(boxcox.transform(data), 3))
+    [[ 0.     0.342]
+     [ 2.068  0.342]
+     [ 3.135  0.417]]
+
+    Notes
+    -----
+    See also QuantileTransformer(output_distribution='uniform') for another
```

Comment:
You could also add a "See also" section so in the online docs a link will be available

Reply:
output uniform creates a gaussian?